crates/sui-core/src/consensus_validator.rs

```diff
@@ -92,7 +103,16 @@ impl TransactionValidator for SuiTxValidator {
         // verify the user transaction signatures as a batch
         obligation
             .verify_all()
-            .wrap_err("Malformed batch (failed to verify)")
+            .wrap_err("Malformed batch (failed to verify)")?;
+
+        // all certificates had valid signatures, schedule them for execution prior to sequencing.
```

Comment:
1. How much latency would this save?
2. We may as well skip processing owned txns in consensus handler?

Reply:
1. I'm not exactly sure, tbh - you probably know better than I.
2. I started to do that and got worried that I might be messing with checkpointing logic in that code path. Interested in @andll's opinion.

Reply:
1. I thought about this a bit. I think this optimization should definitely help reduce contention with increased TPS.
2. We can definitely do that in another PR.

Reply:
Yeah we should keep logic in the consensus handler